tests/test_trailedSources.py

```diff
@@ -145,10 +145,14 @@ def testNaivePlugin(self):
         task.run(catalog, exposure)
         record = catalog[0]
 
+        # Check that root finder converged
+        converged = record.get("ext_trailedSources_Naive_flag_noConverge")
+        self.assertFalse(converged)
+
         # Compare true with measured length and angle
         length = record.get("ext_trailedSources_Naive_length")
         theta = record.get("ext_trailedSources_Naive_angle")
-        self.assertFloatsAlmostEqual(length, self.trail.length, atol=None, rtol=0.5)
+        self.assertFloatsAlmostEqual(length, self.trail.length, atol=None, rtol=0.1)
```

Comment:
make a comment on why you chose that rtol so someone in the future can make a judgment call if things change (like background threshold changes results &c). Something like "we only expect this to be good to 10% so...." or "This should be as accurate as bla"

Reply:
Hmm, I'm actually not sure how to justify these values... Since the measurement relies entirely on previous measurements, I just varied the rtol to find a very rough upper bound to test that the code runs as expected. I've added some comments to clarify.